Eatery/Controllers/EateryTabBarController.swift

```diff
@@ -60,17 +60,18 @@ class EateryTabBarController: UITabBarController {
     ) -> UIInterfaceOrientation {
         .portrait
     }
+    
+    func preselectEatery(withName name: String) {
+        eateriesViewController.preselectEatery(withName: name)
+    }
 
 }
 
 extension EateryTabBarController: UITabBarControllerDelegate {
 
-    func tabBarController(
-        _ tabBarController: UITabBarController,
-        shouldSelect viewController: UIViewController
-    ) -> Bool {
+    func tabBarController(_ tabBarController: UITabBarController, shouldSelect viewController: UIViewController) -> Bool {
         if selectedViewController === viewController,
-            let shared = viewController as? EateriesSharedViewController {
+            let shared = viewController as? CampusEateriesViewController {
```

Comment:
can we rename the `shared` var, seeing as it's no longer an`EateriesSharedViewController`?